notebooks/MAST/exploring_jwst_transmission/comparing_spectral_types_jwst.ipynb

```diff
@@ -0,0 +1,7074 @@
+{
```

Comment:
<p> This cell should use the 'wavrange' variable you set up above</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>